tutorial/EASIER-net Tutorial.ipynb

```diff
@@ -97,6 +123,111 @@
     "mse = np.mean(np.power(predictions - test_y, 2))\n",
     "print(mse)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6ee647bc",
+   "metadata": {},
+   "source": [
+    "##### Classification"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "72c99ff0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def make_class_data(n_samples, num_classes, n_features=20): \n",
+    "    x = np.random.rand(n_samples,n_features)\n",
+    "    y = np.random.randint(low=0, high=num_classes, size=n_samples, dtype=int)\n",
```

Comment:
Can you change this to a logistic regression model? To do this, come up with some fake coefficients:
```
x = np.random.rand(n_samples, n_features)
beta = np.zeros(n_features)
beta[:n_features//2] = 1
prob_y = 1/(1 + np.exp(-np.matmul(x, beta)))
y = np.random.binomial(n=1, p=prob_y)
return x,y
```
I think something like the above would work. I'm only setting half the features to nonzero because easiernet is supposed to be able to do feature selection. ;)

Reply:
thanks this was super helpful! also added a reshape to the y to avoid error when fitting to data